txpipe/map_plots.py

```diff
@@ -0,0 +1,134 @@
+from .data_types import MapsFile, PNGFile
+from .base_stage import PipelineStage
+
+
+class TXMapPlots(PipelineStage):
+    """
+    Make plots of all the available maps.
+    """
+
+    name = "TXMapPlots"
+
+    inputs = [
+        ("source_maps", MapsFile),
```

Comment:
probably don't need to do this now -- but at some point we should have the flexibility to run just subsets of the 2pt  functions and therefore not need to have all the maps

Reply:
Agreed!  I've added a note in twopoint.